cli/enable/enable.go

```diff
@@ -0,0 +1,49 @@
+package enable
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// defaultDirPermissions is rights used to create folders.
+// 0755 - drwxr-xr-x
+// We need to give permission for all to execute
+// read,write for user and only read for others.
+const defaultDirPermissions = 0755
+
+// Enable creates a symbolic link in 'instances_enabled' directory
+// to a script or an application directory.
+func Enable(path string, cliOpts *config.CliOpts) error {
+	var err error
+
+	if _, err := os.Stat(cliOpts.Env.InstancesEnabled); os.IsNotExist(err) {
+		err = os.MkdirAll(cliOpts.Env.InstancesEnabled, defaultDirPermissions)
+		if err != nil {
+			return fmt.Errorf("unable to create %q\n Error: %s",
+				cliOpts.Env.InstancesEnabled, err)
+		}
+		log.Infof("instances enabled directory is created: %q", cliOpts.Env.InstancesEnabled)
+	}
+
+	path, err = filepath.Abs(path)
+	if err != nil {
+		return err
+	}
+
+	pathInfo, err := os.Stat(path)
+	if err != nil {
+		return fmt.Errorf("cannot get info of %q: %s", path, err)
+	}
+
+	if !pathInfo.IsDir() && filepath.Ext(path) != ".lua" {
+		return fmt.Errorf("script %q does not have '.lua' extension", path)
+	}
+
+	return util.CreateSymlink(path,
+		filepath.Join(cliOpts.Env.InstancesEnabled, filepath.Base(path)), true)
```

Comment:
Can we write something about enabled app as a result of command work? I mean, this will look strange if the last message will be "instances enabled dir is created", when I asked to enable application. Maybe something like
"Symbolic link for application "bla-bla-bla" is created in instances enabled directory"
@oleg-jukovec what do you think?

Reply:
Yes, it's better to add a message than to confuse a user.

Reply:
added log message